trigger/project_triggers.go

```diff
@@ -65,6 +65,24 @@ func TriggerDownstreamVersion(args ProcessorArgs) (*model.Version, error) {
 	if err != nil {
 		return nil, err
 	}
+	settings, err := evergreen.GetConfig()
+	if err != nil {
+		return nil, errors.Wrap(err, "error getting evergreen settings")
+	}
+	upstreamProject, err := model.FindOneProjectRef(args.SourceVersion.Identifier)
+	if err != nil {
+		return nil, errors.Wrap(err, "error finding project ref")
+	}
+	for _, module := range proj.Modules {
+		owner, repo := module.GetRepoOwnerAndName()
+		if owner == upstreamProject.Owner && repo == upstreamProject.Repo && module.Branch == upstreamProject.Branch {
+			_, err = repotracker.CreateManifest(*v, proj, upstreamProject.Branch, settings)
+			if err != nil {
+				return nil, err
```

Comment:
errors.WithStack(err)  (do i spend too much time listening to Sam)

Reply:
I actually disagree with Sam on these usually. For errors that are shown to users, they should be wrapped to give enough context, but not wrapped too much (and should never have a stack). For ones that aren't (this one is not), it should be wrapped enough to figure out where it's coming from and maybe have a stack if there are many callers. The function that can produce this error already sufficiently wraps it I believe

Reply:
But even if it does sufficiently wrap it, it doesnt really hurt to add WithStack just in case: if the CreateManifest is ever called from multiple places we want to know where it was called from. (I agree with user facing errors.)

Reply:
fine :(